catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/CachingFederationStrategyTest.java

```diff
@@ -85,39 +90,35 @@
 
   private ExecutorService cacheExecutor, queryExecutor;
 
-  private Query mockQuery;
+  @Mock private Query mockQuery;
 
   private CachingFederationStrategy strategy;
 
-  private PreFederatedQueryPlugin preQueryPlugin;
+  private CachingFederationStrategy federateStrategy;
 
-  private MetacardImpl mockMetacard;
-
-  private Result mockResult;
+  @Mock private PreFederatedQueryPlugin preQueryPlugin;
 
-  private SourceResponse mockResponse;
+  private MetacardImpl mockMetacard;
 
-  private List<Result> results;
+  @Mock private SourceResponse mockResponse;
 
-  private SortedQueryMonitorFactory mockSortedQueryMonitorFactory;
+  @Mock private SortedQueryMonitorFactory mockSortedQueryMonitorFactory;
 
-  private SortedQueryMonitor mockSortedQueryMonitor;
+  @Mock private SortedQueryMonitor mockSortedQueryMonitor;
 
-  private SolrClientAdaptor clientAdaptor;
+  @Mock private SolrClientAdaptor clientAdaptor;
```

Comment:
✏️  Put `mock` in front of the name of all variables that are mocks. It makes it a bit easier to read the tests. I'm not going to comment on every variable that needs it, but there's a few in here.